package.nls.json

```diff
@@ -12,9 +12,14 @@
     "extension.pqtest.config.externals.msbuildPath.description": "Local path to msbuild.exe installation folder.",
     "extension.pqtest.config.externals.nugetPath.description": "Local path to nuget.exe installation folder.",
     "extension.pqtest.config.externals.nugetFeed.description": "Suggested nuget feed URL.",
+    "extension.pqtest.config.externals.versionTag.description": "The version tag of the PQ Sdk tools to be seized.",
```

Comment:
using the term "downloaded" is better/more common than "seized" when referring to the nuget packages.